packages/css-data/src/__generated__/property-value-descriptions.ts

```diff
@@ -0,0 +1,406 @@
+// This file was auto-generated
+export const properties = {
```

Comment:
seems like there should be many more, what do I miss?

Reply:
I only generated a few to show you an example. Will generate them all if you like the result. 

By the way I am regenerating these few in a sec and pushing the new ver.